src/core/currency/wallet/currency-wallet-export.js

```diff
@@ -15,10 +16,16 @@ export function dateFilter(
   return false
 }
 
+export type SearchStringFilterContext = {
+  currencyState: CurrencyState
+}
+
 export function searchStringFilter(
+  context: SearchStringFilterContext,
```

Comment:
If a function needs access to state inside the core, we usually pass an initial `ai: ApiInput` parameter, which contains redux state + dispatch.

However, in this case, it might make more sense to just pass the wallet. The only places where we call `searchStringFilter` as wallet-specific, and the wallet should have everything we could ever need to interpret the transaction.

Either option would be fine.

Reply:
I went with `ai: ApiInput` only because I don't know where the wallet is stored in the data model.